.github/workflows/main.yml

```diff
@@ -76,18 +76,23 @@ jobs:
           go_version: ${{ needs.go-versions.outputs.unstable }}
     steps:
     - uses: actions/checkout@v4
+    - name: Set cores to get stored as "core"
+      run:  sudo bash -c 'echo "core" > /proc/sys/kernel/core_pattern'
     - name: Run tests
       run: make test-containers-test "CEPH_VERSION=${{ matrix.ceph_version }}" "GO_VERSION=${{ matrix.go_version }}" "RESULTS_DIR=$PWD/_results"
     - name: Clean up test containers
+      if: always()
       run: make test-containers-clean "CEPH_VERSION=${{ matrix.ceph_version }}"
     - name: Archive test results
+      if: always()
       uses: actions/upload-artifact@v3
       with:
         name: "go-ceph-results-${{ matrix.ceph_version }}-${{ matrix.go_version }}"
         path: |
           _results/
         retention-days: 30
     - name: Check API Versions and Aging
+      if: always()
```

Comment:
I'm unsure if it's worthwhile to run the API versions check even if the tests fail. The other `always()`s seem to make sense to me though. What do you think about leaving this one off?

Reply:
I also had this thought on whether we should run the API versions check in case of test failure. May be we shouldn't?

Reply:
I was also contemplating it, but my conclusion was: it doesn't hurt to run it, and it just gives additional orthogonal information. It might in some cases avoid another CI run because you see already that there is not only something wrong with the code but you _also_ forgot to update the API docs. So if you don't object, I would leave it in.